### DISCUSSION TITLE
model direction accuracy

### DISCUSSION THREAD

**Discussion by minion50000** [mltrev23/snp500_stock_prediction#16] [Q&A] [answered]
how to improve direction accuracy of model?

**Comment by mltrev23** [comment, marked as answer]
imo we can use custom loss function and metrics for training the model

**Reply by minion50000** [reply]
how to calculate direction accuracy?
and how to create dataset for training?

**Comment by mltrev23** [comment]
according to the direction of previous price value and next value, we can give 1 for up and -1 or 0 for downing.

**Reply by minion50000** [reply]
In that case, back propagation doesn't work well.
